test/setting-integration.test.js

```diff
@@ -43,7 +46,7 @@ describe('Settings Store Integration Tests', async () => {
       caught = err;
     }
     assert.ok(caught);
-  });
+  }).timeout(TIMEOUT_5_SEC);
```

Comment:
FWIW, you can do this at the `describe` level to save yourself needing to change every test's timeout:

```
describe('Nice test!', function () {
  this.timeout(TIMEOUT_5_SEC);
});
```

Reply:
Yeah, it's that balance of having eslint whine because of the unnamed function vs repeating the timeout setting :-/